cmp/53fe1e2b-6a1e-4619-8da2-be8f3b9ab267/component.lp

```diff
@@ -0,0 +1,30 @@
+(librepcb_component 53fe1e2b-6a1e-4619-8da2-be8f3b9ab267
+ (name "Thermistor")
+ (description "")
+ (keywords "")
+ (author "J. R. Schmid")
+ (version "0.1")
+ (created 2020-05-10T13:35:28Z)
+ (deprecated false)
+ (category 1039f038-20a6-4bfe-89c1-99f34fbb45bd)
+ (schematic_only false)
+ (default_value "{{PARTNUMBER or DEVICE}}")
+ (prefix "TH")
+ (attribute "RESISTANCE" (type resistance) (unit kiloohm) (value ""))
+ (signal 6996c9c3-b8d9-4538-a00c-07899c55dd5f (name "2") (role passive)
+  (required false) (negated false) (clock false) (forced_net "")
+ )
+ (signal 14650991-6a1f-4106-82e9-4418f24f27d5 (name "1") (role passive)
+  (required false) (negated false) (clock false) (forced_net "")
+ )
+ (variant 098e4af0-ed0e-4dd3-91ce-3a057b31e382 (norm "")
+  (name "default")
+  (description "")
```

Comment:
Since this is the European / International variant of the symbol, we can specify that (and maybe in the future add an US variant of the symbol).

```suggestion
 (variant 098e4af0-ed0e-4dd3-91ce-3a057b31e382 (norm "IEC 60617")
  (name "European")
  (description "")
```

Reply:
Changed.